shared/app/nav.native.js

```diff
@@ -195,7 +195,7 @@ class MainNavStack extends Component<any, {verticalOffset: number}> {
     const content = (
       <Kb.NativeView style={styles.content}>
         {stacks}
-        {![chatTab, peopleTab].includes(props.routeSelected) ? <Offline key="offline" /> : null}
+        {![chatTab, peopleTab, settingsTab].includes(props.routeSelected) ? <Offline key="offline" /> : null}
```

Comment:
As we talked about at standup, my proposal here is to let each item in the Settings menu handle offline mode rather than trying to modify Offline to work well enough. We already have tickets for Teams, Git, and Devices, and then after that we can either make the other things use Reloadable, do nothing (e.g. for Sign out or About), or Offline.